### ISSUE TITLE
Ad-Hoc command result form not shown

### ISSUE THREAD

**Issue by Zash** [conversejs/converse.js#2240]
**Describe the bug** (or feature request?)
Ad-hoc commands can consist of multiple steps, as well as return the final results in a dataform. This does not appear to be supported.

**To Reproduce**
1. Go to Settings (the :gear: button in the control box)
2. Click on Commands and fill in the address of an entity that exposes a multi-step command or one that returns a form as result.
3. Run the command.
4. Observe lack of result form.

**Expected behavior**
If a command returns a result form, the data in it should be shown.

For multi-step commands, the next form should be shown.

**Screenshots**

It just shows the first form, no changes.

**Environment (please complete the following information):**
 - Debian 10 desktop
 - Firefox 68.12.0esr ESR
 - Converse.js version ced43363f626c7b11da418d5f84220a938314903

**Additional context**

The Prosody module [mod_adhoc_dataforms_demo](https://modules.prosody.im/mod_adhoc_dataforms_demo.html) can be used to test multi-step commands.

**Comment by poVoq** [comment]
Just to add: Biboumi requires a multistep ad-hoc command to set up IRC NICKSERV, thus this would probably be a often used feature.

**Comment by poVoq** [comment]
Still an issue on the "configure a few settings for that IRC server" adhoc command of Biboumi.

I just managed to break my Biboumi config by trying to use adhoc commands with ConverseJS.

**Comment by poVoq** [comment]
In the Firefox browser console I get the following error when and only when I press the buggy command mentioned above:

```
ERROR: Error while trying to execute command for irc.example.com log.js:64:19
    log log.js:64
    error log.js:81
    toggleCommandForm utils.js:159
    toggleCommandForm adhoc-commands.js:80
    toggleCommandForm adhoc-commands.js:40
    handleEvent lit-html.ts:2047
    (Async: EventListener.handleEvent)
    _$AI lit-html.ts:2036
    m lit-html.ts:1240
    T lit-html.ts:1582
    _$AI lit-html.ts:1434
    S lit-html.ts:1647
    _$AI lit-html.ts:1438
    m lit-html.ts:1240
    T lit-html.ts:1582
    _$AI lit-html.ts:1434
    m lit-html.ts:1240
    T lit-html.ts:1569
    _$AI lit-html.ts:1434
    tu lit-html.ts:677
    update lit-element.ts:165
    performUpdate reactive-element.ts:1302
    scheduleUpdate reactive-element.ts:1234
    _$E_ reactive-element.ts:1206
    requestUpdate reactive-element.ts:1181
    set reactive-element.ts:707
    fetchCommands adhoc-commands.js:61
    fetchCommands adhoc-commands.js:36
    handleEvent lit-html.ts:2047
    (Async: EventListener.handleEvent)
    _$AI lit-html.ts:2036
    m lit-html.ts:1240
    T lit-html.ts:1582
    _$AI lit-html.ts:1434
    tu lit-html.ts:677
    update lit-element.ts:165
    performUpdate reactive-element.ts:1302
    scheduleUpdate reactive-element.ts:1234
    _$E_ reactive-element.ts:1206
    requestUpdate reactive-element.ts:1181
    o reactive-element.ts:923
    Nd reactive-element.ts:906
    _u lit-element.ts:115
    hv element.js:5
    <anonymous> adhoc-commands.js:25
    p lit-html.ts:1181
    T lit-html.ts:1572
    _$AI lit-html.ts:1434
    tu lit-html.ts:677
    render view.js:79
    initialize base.js:28
    initialize user-settings.js:11
    Iv view.js:45
    s helpers.js:32
    s helpers.js:32
    create api.js:46
    show api.js:26
    showUserSettingsModal statusview.js:32
    handleEvent lit-html.ts:2047
```

**Comment by poVoq** [comment]
Actually the Biboumi issue I have mentioned above seems to be fixed in 10.0.0.

I was still testing with an outdated 9.1.1. without noticing.
